pkg/discovery/network/ovnkubernetes.go

```diff
@@ -0,0 +1,62 @@
+package network
+
+import (
+	"fmt"
+	"strings"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	ovnKubeService = "ovnkube-db"
+	OvnNBDB        = "OVN_NBDB"
+	OvnSBDB        = "OVN_SBDB"
+)
+
+func discoverOvnKubernetesNetwork(clientSet kubernetes.Interface) (*ClusterNetwork, error) {
+	ovnDbPod, err := findPod(clientSet, "name=ovnkube-db")
+
+	if err != nil || ovnDbPod == nil {
+		return nil, err
+	}
+
+	if _, err := clientSet.CoreV1().Services(ovnDbPod.Namespace).Get(ovnKubeService, v1.GetOptions{}); err != nil {
+		return nil, fmt.Errorf("error finding %q service in %q namespace", ovnKubeService, ovnDbPod.Namespace)
+	}
+
+	dbConnectionProtocol := "tcp"
+
+	for _, container := range ovnDbPod.Spec.Containers {
+		for _, envVar := range container.Env {
+			if envVar.Name == "OVN_SSL_ENABLE" {
+				if strings.ToUpper(envVar.Value) != "NO" {
+					dbConnectionProtocol = "ssl"
+				}
+			}
+		}
+	}
+
+	dbConnection := fmt.Sprintf("%s://%s.%s", dbConnectionProtocol, ovnKubeService, ovnDbPod.Namespace)
+
+	clusterNetwork := &ClusterNetwork{
+		NetworkPlugin: "OVNKubernetes",
+		PluginSettings: map[string]string{
+			OvnNBDB: dbConnection + ":6641",
```

Comment:
Shouldn't we use a constant for these?

Reply:
We don't replay them in more places, for OVNKubernetes yes but we talked about adding all those to admiral as a common place (it's operator + subm depending on those), also the default port for openshift is different, but that one is detected with the openshift cluster network operator.

Reply:
Ok, converted them to constants, no kitties were harmed :D :-P :)